tests/admin_changelist/models.py

```diff
@@ -19,6 +19,12 @@ class Child(models.Model):
     age = models.IntegerField(null=True, blank=True)
 
 
+class GrandChild(models.Model):
+    parent = models.ForeignKey(Child, models.SET_NULL, editable=False, null=True)
+    name = models.CharField(max_length=30, blank=True)
+    age = models.IntegerField(null=True, blank=True)
```

Comment:
I would remove `age` in this model, I don't think it adds anything valuable to the tests. What do you think?

Reply:
Yes, makes sense, this was added from the original PR and it doesn't seem necessary.